.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+To report a good issue, please verify that you validate these points:
+
+- [ ] you read the [documentation](../../#available-commands--options) and the [some advanced use cases](../../#more-advanced-use-cases)
+- [ ] upgrade and test with the last version
+- [ ] specify the version you are using
+- [ ] give us the maximum of information that could help (what you want to do, what you did and what happens)
```

Comment:
> give us the maximum of information that could help (what you want to do, what you did and what happens)

or "include steps you followed to reproduce the problem"? it'd also be nice to ask for a gist with the output of the problematic command when it's run with `--debug`.